tests/device/overlay_generator_channels_test.py

```
@@ -0,0 +1,126 @@
+            ImageDraw.Draw(img).rectangle([10, 10, 86, 86], fill=(255, 255, 255, 255))
+            img.save(icons / f"{key}.png")
+
+        lines = ["app: rt", "match: [rt]", "output: rt_template",
+                 "icon_dir: icons", "mode: alpha", "bindings:"]
+        for (mod, mods), key in zip(VARIANTS, KEYS):
+            lines.append(f"  - {{ key: {key}, mods: {mods}, "
+                         f"icon: {key}.png, label: {mod.name} }}")
+        (tmp / "b.yaml").write_text("\n".join(lines) + "\n")
+
+        cls.out = tmp / "out"
+        cls.proc = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)